SocialFilter/sections/specific.txt

```diff
@@ -53,6 +53,7 @@ ansa.it##.social-menu li.linkedin
 aktuality.sk##a[href^="https://www.facebook.com/sharer.php"]
 dila.ua##.article-share-block
 enginform.com##.social_block
+m-timesofindia-com.cdn.ampproject.org##div.as_heading > .pdTop_2 > div[id^="ss-"]
```

Comment:
It seems that this rule doesn't hide social widgets from the next article (second screenshot from the report), because it's in a shadow-root.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/102797589-30628480-43b0-11eb-89de-39c92333317e.png)

</details>

So I guess we also could add something like this:
```
m-timesofindia-com.cdn.ampproject.org#%#//scriptlet('hide-in-shadow-dom', 'div.as_heading > .pdTop_2 > div[id^="ss-"]')
```

If I'm not wrong, there is also a not blocked ad leftover.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/102797667-4d975300-43b0-11eb-9efb-f77ec5fc5965.png)

</details>

Reply:
That's strange, i am not getting soc buttons on the next item.

Reply:
I see them only with apps, with extensions it's blocked.
Do you see shadow-root in source code (like in my first screenshot)?